wallets/cln/client.js

```diff
@@ -1 +1,39 @@
+import { fetchWithTimeout } from '@/lib/fetch'
+import { assertContentTypeJson, assertResponseOk } from '@/lib/url'
+
 export * from '@/wallets/cln'
+
+export async function testSendPayment ({ socket, runePay }, { logger, signal }) {
+  // TODO: check rune
+}
+
+export async function sendPayment (bolt11, { socket, runePay }, { logger, signal }) {
+  const url = `https://${socket}/v1/pay`
```

Comment:
You can't hardcode the protocol since we allow HTTP over Tor.

This also breaks testing in local dev so I couldn't test this.

> So this will definitely require testing with a real CLNRest server.

We have a CLN node in our docker-compose setup that you should be able to test with.